TAScheduler/AcceptanceTests/AssignTAtoLab tests.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+
+from django.test import TestCase
+
+from TAScheduler.models import User, TA, Course, TAToCourse
+
+
+#
+class SuccessfulAssignment(TestCase):
+    user = None
+    ta = None
+    course = None
+    tatocourse = None
+
+    def setUp(self):
```

Comment:
Whenever you create an object in the database, I think you have to save it in order for it to actually be added for these tests. 
Rock's example:    
temp = MyUser(name=i, password=i)
temp.save()